lua/downloader/cache_load.lua

```diff
@@ -19,6 +18,17 @@ function MODULE:Run(context)
 
             context.ignoreResources[addon.wsid] = true
             context.gamemodeAddons[addon.wsid] = scanned.isGamemode
+            context.manualAddons[addon.wsid] = scanned.manual
+
+            if scanned.scanResult then -- Compatibility with older caches
+                context.scanResult[addon.wsid] = scanned.scanResult
+            else
+                if scanned.hasResource then
+                    context.scanResult[addon.wsid] = { selected = true, type = "Resources" }
+                else
+                    context.scanResult[addon.wsid] = { selected = false, type = "Lua" }
+                end
+            end
```

Comment:
```suggestion
            elseif scanned.hasResource then
                context.scanResult[addon.wsid] = { selected = true, type = "Resources" }
            else
                context.scanResult[addon.wsid] = { selected = false, type = "Lua" }
            end
```

Reply:
OH NO, IT'S THE CODING POLICE!!

Reply:
Btw the previous "if" was removed by ed289f8 and I merged these lines into the same check in a block right above:
![image](https://user-images.githubusercontent.com/5098527/234171556-4fd82b0c-8a8a-40b8-a972-20052b9c8479.png)